### PR TITLE
Add Points section - TEDU 129

### PR DESCRIPTION
**Pull request by favour-chibueze**
TEDU- 129

<a href="https://gitpod.io/#https://github.com/mautic/user-documentation/pull/112"></a>

### CODE REVIEW

**Comment by RCheesley** [comment]
Please fix the Vale errors in this PR

File: docs/points/points.rst [hunk 1]
```diff
@@ -3,4 +3,99 @@
...
+================
+
+**Modify Contact’s Segments** - Add a Contact to or remove a Contact from any Segments you have published.
+
+**Modify Contact’s tags** - Add or remove any tags on the Contact record. If a tag doesn't exist, you may create a new one in the edit window for this event.
+
+Add-on triggers
+===============
+
+**Push Contact to Integration** - To only push Contacts to an Integration after hitting a minimum point total, use this option. You must have the **Triggered action push Contacts to Integration** option selected in the Integration. After selecting this event, the system displays a dialog box where you can choose which Integration to push the Contact to. For example, if your definition of MQL is based on point values, you may hold non-MQLs out of your CRM. Once a Contact meets the Points requirement to be an MQL, push the Contact to your CRM.
```
Comment (RCheesley) [inline docs/points/points.rst:72]:
```suggestion
**Push Contact to Integration** - To only push Contacts to an Integration after hitting a minimum point total, use this option. You must have the **Triggered action push Contacts to Integration** option selected in the Integration. After selecting this event, the system displays a dialog box where you can choose which Integration to push the Contact to. For example, if your definition of a Marketing Qualified Lead (MQL) is based on point values, you may decide to only push contacts who are MQLs to your CRM. Once a Contact meets the Points requirement to be an MQL, use this action to push the Contact to your CRM.
```

File: docs/points/points.rst [hunk 2]
```diff
@@ -3,4 +3,99 @@
 Points
 ######
 
-.. vale on
\ No newline at end of file
+.. vale on
+
+Points provide a way for properly weighing Contacts. These Points have both triggers and actions. Each term is properly defined and a thorough understanding of how Points function ensures that your overall marketing automation process is successful.
```
Comment (RCheesley) [inline docs/points/points.rst:8]:
```suggestion
Points provide a way for properly weighing Contacts. These Points have both triggers and actions. Each term is defined below, and a thorough understanding of how Points function ensures that your overall marketing automation process is successful.
```

File: docs/points/points.rst [hunk 3]
```diff
@@ -3,4 +3,99 @@
...
+
+Point actions
+*************
+
+Point actions are those times when a Contact receives a change in their point total. These actions can be either positive or negative point changes and are based on a particular action as you determine.
```
Comment (RCheesley) [inline docs/points/points.rst:13]:
```suggestion
Point actions are those times when a Contact receives a change in their Point total. These actions can be either positive or negative Point changes and occur based on a particular action as you determine.
```

File: docs/points/points.rst [hunk 4]
```diff
@@ -3,4 +3,99 @@
...
+
+2. In the main panel, there are four boxes for key information. Enter the appropriate information.
+
+   * **Name** - The name of your action. This is how the action displays in your list of actions, so choose an identifiable name.
+
+   * **Description** - Add a description to help you find certain actions. There may be more actions which are similar or more in-depth.
+
+   * **Change Points (+/-)** - The value change to set for the action. The ``+`` isn't necessary when adding Points. When subtracting Points, add the ``-`` symbol.
+
+   * **When a Contact…** - This is the behavior or action the Contact must complete to receive the Points.
```
Comment (RCheesley) [inline docs/points/points.rst:30]:
```suggestion
   * **When a Contact…** - This is the behavior or action the Contact must complete for the action to be triggered.
```

Comment (RCheesley) [reply docs/points/points.rst:30]: We don't want to assume that points are being added, as they can equally be subtracted.

File: docs/points/points.rst [hunk 5]
```diff
@@ -3,4 +3,99 @@
...
+
+3. On the right side of the page is more information:
+
+   * **Category** - Organize your point actions based on their goals, Campaigns, etc. For more information, see :doc:`Categories</categories/categories-overview>`. All Points accumulate on a Contact record, regardless of Category. There is one score or Points model for each Contact.
```
Comment (RCheesley) [inline docs/points/points.rst:34]:
```suggestion
   * **Category** - Organize your Point actions based on their goals, Campaigns, etc. For more information, see :doc:`Categories</categories/categories-overview>`. All Points accumulate on a Contact record, regardless of Category. There is one Points score for each Contact.
```

**Review by RCheesley — CHANGES_REQUESTED** [review]
A lot of issues flagged by Vale and some corporate references to be removed. Please review your PRs on Gitpod once they are created - it will show you all the errors if they are not showing locally for you.

File: docs/points/points.rst (hunk 4 above)
Comment (favour-chibueze) [reply docs/points/points.rst:30]: Got it

File: docs/points/points_troubleshooting.rst [hunk 6]
```diff
@@ -1,6 +1,15 @@
-.. vale off
-
 Points troubleshooting
```
Comment (RCheesley) [inline docs/points/points_troubleshooting.rst:1]: Do we still need to have this as a page heading? I think we do - not sure why it is removed here.

File: docs/points/points_troubleshooting.rst [hunk 7]
```diff
@@ -1,6 +1,15 @@
...
+
+To workaround this issue, try one of the following options:
+
+1. Make sure that you aren't testing the page visit while logged into Mautic. Mautic ignores activity from Mautic Administrators. So, it's suggested that you use an anonymous session, an incognito window, another browser, or log out of Mautic.
+
+2. The tracking of Point actions is currently done once per Contact. This means that subsequent visits won't re-trigger the action if already triggered once.
+
+3. Ensure that the URL defined either matches exactly the URL visited or use a wildcard. A URL can include the schema, host/domain, path, query parameters, and/or fragment.
+
+For example, if you have a URL of ``https://example.com`` and the page hit registers as ``https://example.com/index.php?foo=bar``, the point action won't be recognized.. However, if you use ``https://example.com*`` as the URL, it matches the rule and thus gets triggered.
```
Comment (RCheesley) [inline docs/points/points_troubleshooting.rst:15]:
```suggestion
For example, if you have a URL of ``https://example.com`` and the page hit registers as ``https://example.com/index.php?foo=bar``, the point action won't be recognized. However, if you use ``https://example.com*`` as the URL, it matches the rule and thus gets triggered.
```

File: docs/points/points_troubleshooting.rst (hunk 6 above)
Comment (favour-chibueze) [reply docs/points/points_troubleshooting.rst:1]: Yes, we do but what was removed was the vale warning which wasn't needed

**Comment by favour-chibueze** [comment]
@RCheesley

File: docs/points/points.rst [hunk 8]
```diff
@@ -3,4 +3,92 @@
...
+
+Email triggers
+==============
+
+**Send an Email** - Send a template Email to the Contact based on their engagement. This may be some sort of special offer, congratulations, etc.
+
+.. image:: images/send-an-email-to-user.png
+    :alt: Screenshot of Send an Email to user Email trigger
+
+**Send an Email to user** - Tell a team member that a Contact has reached a minimum number of Points. There is an option in this event to send the Email to the Contact’s owner. You may either write a basic Email in the editor, or use a template Email.
```
Comment (RCheesley) [inline docs/points/points.rst:86]:
```suggestion
**Send an Email to User** - Tell a team member that a Contact has reached a minimum number of Points. There is an option in this event to send the Email to the Contact's owner. You may either write a basic Email in the editor, or use a template Email.
```

File: docs/points/points.rst [hunk 9]
```diff
@@ -3,4 +3,92 @@
...
+
+    The Push Contact to Integration action isn't supported with the Salesforce Plugin.
+
+Email triggers
+==============
+
+**Send an Email** - Send a template Email to the Contact based on their engagement. This may be some sort of special offer, congratulations, etc.
+
+.. image:: images/send-an-email-to-user.png
+    :alt: Screenshot of Send an Email to user Email trigger
```
Comment (RCheesley) [inline docs/points/points.rst:84]:
```suggestion
    :alt: Screenshot of Send an Email to User Email trigger
```

File: docs/points/points.rst (hunk 8 above)
Comment (RCheesley) [reply docs/points/points.rst:86]: Also these bullet points look visually strange, with a grey background:

screenshot-favourchibu-userdocumen-uc1777detgv ws-eu78 gitpod io-2022 12 09-11_01_53

**Review by RCheesley — CHANGES_REQUESTED** [review]
Not sure if you checked Vale for this one as there were quite a few to fix.

Also the last set of bullet points looks a bit strange - maybe something to do with the indentation?

Also corrected some apostrophes - be careful when pasting to import as they are using the wrong type which VSCode flags up by making them red.

**Comment by favour-chibueze** [comment]
@RCheesley please check this PR

**Comment by favour-chibueze** [comment]
@RCheesley This hasn't been reviewed

**Comment by RCheesley** [comment]
One tiny change which I just committed directly so we can go straight to merge 🚀

**Pull request merged**